gtest/src/system.rs

```diff
@@ -79,19 +85,34 @@ impl System {
         self.0.borrow_mut().run_dispatch(dispatch)
     }
 
-    pub fn spend_blocks(&self, amount: u32) {
+    pub fn send_delayed_dispatch(&self, delay: u32, dispatch: Dispatch) {
```

Comment:
users and external api aren't allowed to send delayed messages. it's program-only functionality
```suggestion
```